jina/proto/message.py

```diff
@@ -0,0 +1,364 @@
+import os
+import sys
+from typing import List, Union, Optional
+
+from . import jina_pb2
+from .. import __version__, __proto_version__
+from ..enums import CompressAlgo
+from ..excepts import MismatchedVersion
+from ..logging import default_logger
+
+_trigger_body_fields = set(kk
+                           for v in [jina_pb2.Request.IndexRequest,
+                                     jina_pb2.Request.SearchRequest,
+                                     jina_pb2.Request.TrainRequest,
+                                     jina_pb2.Request.ControlRequest]
+                           for kk in v.DESCRIPTOR.fields_by_name.keys())
+
+_trigger_req_fields = set(jina_pb2.Request.DESCRIPTOR.fields_by_name.keys()).difference(
+    {'train', 'index', 'search', 'control'})
+
+_trigger_fields = _trigger_req_fields.union(_trigger_body_fields)
+_empty_request = jina_pb2.Request()
+
+__all__ = ['ProtoMessage', 'ControlMessage']
+
+
+class LazyRequest:
```

Comment:
can't we split this into another file? I think is better to keep smaller self-contained files